src/bundler.spec.ts

```diff
@@ -0,0 +1,63 @@
+import * as tar from 'tar';
+import * as fs from 'fs';
+import * as asyncHelpers from './async.helpers';
+import { Bundler } from './bundler';
+import * as fileHelpers from './file.helpers';
+
+describe('Bundler', () => {
+  const writeFileAsync = jest.spyOn(asyncHelpers, 'writeFileAsync').mockResolvedValue(undefined);
+  const tarC = jest.spyOn(tar, 'c').mockResolvedValue(undefined as never);
```

Comment:
So the value should be `never` `undefined`? 🤔 😜 🥁

Reply:
Yeah, this is really weird. Mock typings for Jest apparently always take the last overload of function, which in this case has a return type of `never`... 🤷‍♂️